README.md

```diff
@@ -29,8 +29,6 @@ because these components attempt to connect to Cassandra right away and hard fai
 Your Agent hostname is `jaeger-agent.${NAMESPACE}.svc.cluster.local`
 
 Once everything is ready, `kubectl get service jaeger-query` tells you where to find Jaeger URL.
-If you are using `minikube` to setup your Kubernetes cluster, the command `minikube service jaeger-query --url`
```

Comment:
removed, The same is in dev demplate section and production template will not be probably used with minikube.